sdk/storage/azure-storage-blob/azure/storage/blob/_shared/request_handlers.py

```diff
@@ -46,10 +41,10 @@ def serialize_iso(attr):
         return date + 'Z'
     except (ValueError, OverflowError) as err:
         msg = "Unable to serialize datetime object."
-        raise_with_traceback(ValueError, msg, err)
+        raise (ValueError, msg, err)  # pylint: disable=raising-bad-type
```

Comment:
Not sure what this syntax is? Is that `raise` as a function? I think this is maybe some old Python 2 stuff?

Can we try changing to this? I think this should be functionally equivalent.
`raise ValueError("Unable to serialize datetime object.") from err`

I also think if you fix both of these you won't need to disable `inconsistent-return-statements` at the top of this function.

Reply:
So I think I originally tried that, and this is the result: 
`azure\storage\blob\_shared\request_handlers.py:44: [E0702(raising-bad-type), serialize_iso] Raising tuple while only classes or instances are allowed`

So I think we will have to just leave the disable in 😢